.github/workflows/pythonpackage.yml

```diff
@@ -151,7 +151,7 @@ jobs:
         python -m pip install "setuptools ~= 42.0" wheel "setuptools_scm[toml] >= 3.4" \
                               setuptools_scm_git_archive "pip>=10.0.1"
         python setup.py develop
-        INSTALLED_VERSION=$(python -c 'import templateflow as tf; print(tf.__version__, end="")')
+        INSTALLED_VERSION=$(python -c 'import templateflow as tf; print(tf.__version__, end="")' | tail -1)
```

Comment:
What's this tail for?

Reply:
the tails ensure we grab the last line only (the version info)

it is filtering out the message generated when the skeleton is generated:

```
TEMPLATEFLOW_HOME directory (S3 type) was up-to-date.
```